packages/web-components/src/components/masthead/masthead-l1.ts

```diff
@@ -200,11 +630,103 @@ class DDSMastheadL1 extends StableSelectorMixin(LitElement) {
    *
    * @param {PointerEvent} event The click event from the user
    */
-  protected static _toggleMobileSubsection(event: PointerEvent) {
+  protected _toggleSubsection(event: PointerEvent) {
+    const { isMobileVersion } = this;
     const { currentTarget } = event;
     const button = currentTarget as HTMLElement;
-    button.classList.toggle('is-open');
-    button.nextElementSibling?.toggleAttribute('hidden');
+    const dropdown = button.nextElementSibling as HTMLElement;
+    const isOpen = dropdown.classList.contains('is-open');
+
+    if (!isMobileVersion && dropdown && !isOpen) {
+      // Get Button & Dropdown locations & widths
+      const buttonRect = button.getBoundingClientRect().toJSON();
+      const dropdownRect = dropdown.getBoundingClientRect().toJSON();
+      const viewportWidth = window.innerWidth;
+      const isLTR =
+        window.getComputedStyle(this).direction.toUpperCase() === 'LTR';
+      const dirMultiplier = isLTR ? -1 : 1;
+
+      // Set logical bounding client rect properties
+      buttonRect.inlineStart = isLTR
+        ? buttonRect.left
+        : viewportWidth - buttonRect.right;
+      buttonRect.inlineEnd = isLTR
+        ? buttonRect.right
+        : viewportWidth - buttonRect.left;
+      dropdownRect.inlineStart = isLTR
+        ? dropdownRect.left
+        : viewportWidth - dropdownRect.right;
+      dropdownRect.inlineEnd = isLTR
+        ? dropdownRect.right
+        : viewportWidth - dropdownRect.left;
+
+      // What would the left/right boundaries be if we centered the dropdown?
+      const startBoundaryIfCentered = (viewportWidth - dropdownRect.width) / 2;
+      const endBoundaryIfCentered =
+        startBoundaryIfCentered + dropdownRect.width;
+
+      // If start-aligning to button doesn't overflow viewport.
+      if (buttonRect.inlineStart + dropdownRect.width < viewportWidth) {
+        // Do nothing, the styles are fine.
+      }
```

Comment:
Doesn't look like you need this.

Reply:
This is needed to prevent landing in the `else if` and `else` blocks.